auto_remediate/security_hub_rules.py

```diff
@@ -602,6 +602,30 @@ def iam_user_unused_credentials_check(self, resource_id):
 
             return True
 
+    def mfa_enabled_for_iam_console_access(self, resource_id):
+        """Deletes login profile for user with console access that doesn't have MFA enabled
+
+        Arguments:
+            resource_id {string} -- IAM User ID
+
+        Returns:
+            boolean -- True if remediation is successful
+        """
+        try:
+            page_user = self.client_iam.get_paginator("list_users").paginate()
+            for username in page_user.search(
+                f"Users[?UserId == '{resource_id}'].UserName"
+            ):
+                self.client_iam.delete_login_profile(UserName=username)
+                self.logging.info(f"Deleted login profile for user {username}.")
+                return True
+        except:
+            self.logging.error(
```

Comment:
Change to: `self.logging.error(f"Could not delete login profile for IAM User '{resource_id}'.")`

Reply:
8335333a412075e2efc6eb2bdc9621eafa096771